### ISSUE TITLE
Вопрос по 2 тестированию

### ISSUE THREAD

**Issue by Avonna** [AnikeychikPG/AnikeychikPG-VKR#1]
Здравствуйте.  
Прошу прощения за беспокойство. Я сейчас обучаюсь на курсе Data Science в МГТУ и прохожу второе тестирование. Там есть задание с датасетом: #Разработать и обучить нейронную сеть на TensorFlow для решения задачи классификации рукописных цифр. Набор данных: MNIST, источник: from keras.datasets import mnist 
 
#В нейронной сети должно быть 3 Dense-слоя: два скрытых слоя, состоящих из 128 нейронов каждый, и выходной слой. Активационные функции на скрытых слоях relu, на выходном - softmax. 
 
#В качестве ответа необходимо указать количество оптимизируемых параметров (Trainable params) в виде целого числа 
#и прикрепить к ответу подробное решение с кодом в виде файла или архива с файлами.

Хотела спросить, вдруг у вас осталось решение. Правильный ли ответ: Total params: 147,402
Trainable params: 147,402
Non-trainable params: 0
__________________________Благодарю.

**Comment by AnikeychikPG** [comment]
Здравствуйте, у нас было такое же задание, но дата сет из титаника☹️
________________________________
От: Avona ***@***.***>
Отправлено: Saturday, June 11, 2022 1:17:25 PM
Кому: AnikeychikPG/AnikeychikPG-VKR ***@***.***>
Копия: Subscribed ***@***.***>
Тема: [AnikeychikPG/AnikeychikPG-VKR] Вопрос по 2 тестированию (Issue #1)


Здравствуйте.
Прошу прощения за беспокойство. Я сейчас обучаюсь на курсе Data Science в МГТУ и прохожу второе тестирование. Там есть задание с датасетом: #Разработать и обучить нейронную сеть на TensorFlow для решения задачи классификации рукописных цифр. Набор данных: MNIST, источник: from keras.datasets import mnist

#В нейронной сети должно быть 3 Dense-слоя: два скрытых слоя, состоящих из 128 нейронов каждый, и выходной слой. Активационные функции на скрытых слоях relu, на выходном - softmax.

#В качестве ответа необходимо указать количество оптимизируемых параметров (Trainable params) в виде целого числа
#и прикрепить к ответу подробное решение с кодом в виде файла или архива с файлами.

Хотела спросить, вдруг у вас осталось решение. Правильный ли ответ: Total params: 147,402
Trainable params: 147,402
Non-trainable params: 0
__________________________Благодарю.

**Comment by AnikeychikPG** [comment]
+79853319571 пишите в телегу если что
________________________________
От: Аникейчик Павел ***@***.***>
Отправлено: Saturday, June 11, 2022 1:47:19 PM
Кому: AnikeychikPG/AnikeychikPG-VKR ***@***.***>; AnikeychikPG/AnikeychikPG-VKR ***@***.***>
Копия: Subscribed ***@***.***>
Тема: Re: [AnikeychikPG/AnikeychikPG-VKR] Вопрос по 2 тестированию (Issue #1)

 Здравствуйте, у нас было такое же задание, но дата сет из титаника☹️
________________________________
От: Avona ***@***.***>
Отправлено: Saturday, June 11, 2022 1:17:25 PM
Кому: AnikeychikPG/AnikeychikPG-VKR ***@***.***>
Копия: Subscribed ***@***.***>
Тема: [AnikeychikPG/AnikeychikPG-VKR] Вопрос по 2 тестированию (Issue #1)


Здравствуйте.
Прошу прощения за беспокойство. Я сейчас обучаюсь на курсе Data Science в МГТУ и прохожу второе тестирование. Там есть задание с датасетом: #Разработать и обучить нейронную сеть на TensorFlow для решения задачи классификации рукописных цифр. Набор данных: MNIST, источник: from keras.datasets import mnist

#В нейронной сети должно быть 3 Dense-слоя: два скрытых слоя, состоящих из 128 нейронов каждый, и выходной слой. Активационные функции на скрытых слоях relu, на выходном - softmax.

#В качестве ответа необходимо указать количество оптимизируемых параметров (Trainable params) в виде целого числа
#и прикрепить к ответу подробное решение с кодом в виде файла или архива с файлами.

Хотела спросить, вдруг у вас осталось решение. Правильный ли ответ: Total params: 147,402
Trainable params: 147,402
Non-trainable params: 0
__________________________Благодарю.

**Comment by Avonna** [comment]
Спасибо большое. Нашла свою ошибку. Не правильный ответ был. Просто спокойно посмотрела код, увидела, где ошиблась и все нормально. Благодарю за помощь. В телегу напишу, если Вы не против, задружимся на почве любви к Data Science.
